### PR TITLE
Phase 2 - Add dockerfile

### PR DESCRIPTION
**Pull request by ninochang**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
 
 ###
 
 FROM        ${base} as build
 
-ARG         version=
-ARG         repo=
+# ARG         version=
+# ARG         repo=
+# ARG         transformer_model=
```
Comment (allanlei) [inline Dockerfile:10]: Remove

**Review by allanlei — CHANGES_REQUESTED** [review]
Change to poetry

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
 
 FROM        ${base}
 
+ARG         ckip_transformer_model=bert-base
+
+ENV         PYTHONUNBUFFERED=1
+ENV         CKIP_TRANSFORMER_MODEL=${ckip_transformer_model}
+
+EXPOSE      8000/tcp
+CMD         ["uwsgi", "--ini", "config/uwsgi.ini", "--http", ":8000"]
```
Comment (allanlei) [inline Dockerfile:25]: Where's the `uwsgi` dependency?

**Review by allanlei — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
-ARG         repo=
+WORKDIR     /usr/src/app
+COPY        requirements.txt .
 
-RUN         apk add --no-cache --virtual .build-deps \
-                build-base && \
-            wget -O - https://github.com/${repo}/archive/refs/tags/v${version}.tar.gz | tar xz
+RUN         apt-get update && \
+            apt-get install -y build-essential && \
+            pip install -r requirements.txt
```
Comment (allanlei) [inline Dockerfile:13]: There did this file come from?

**Review by allanlei — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (ninochang) [reply Dockerfile:25]: added

File: Dockerfile (hunk 3 above)
Comment (ninochang) [reply Dockerfile:13]: added

**Review by allanlei — APPROVED** [review]

**Review by allanlei — APPROVED** [review]

**Pull request merged**